mm2src/lp_swap/maker_swap.rs

```diff
@@ -349,6 +388,20 @@ impl MakerSwap {
         let maker_coin_swap_contract_address = self.maker_coin.swap_contract_address();
         let taker_coin_swap_contract_address = self.taker_coin.swap_contract_address();
 
+        let maker_coin_htlc_privkey = if self.maker_coin.is_privacy() {
+            let secp_privkey = SecretKey::new(&mut rand6::thread_rng());
```

Comment:
why the use of 2 `rand` dependencies? Does this provide more randomness, or is it something else?

Reply:
Because our other dependencies rely on different rand versions. I think it's easier to do it this way than patch or update deps like secp256k1.